README.md

```diff
@@ -49,7 +49,21 @@ If you'd like to add a new sample to this repository, please follow the below st
     - `thumbnailPath`: The path to the thumbnail image of the sample. This should be the path to the thumbnail image relative to the `repositoryUrl` directory. It is recommended to add the thumbnail to the `.samples/icons` directory and refer it here.
     - `documentationPath`: The path to the documentation/README.md of the sample. This should be the path to the documentation relative to the `repositoryUrl` directory.
     - `tags`: Tags for the sample. This should be a list of strings.
-4. Submit a pull request to this repository.
+
+4. If you're adding a Quick Deployable Sample, follow these additional steps:
+    - Ensure the `componentType` is one of the following:
+        - `service`
+        - `web-application`
+        - `manual-task`
+        - `scheduled-task`
+    - Ensure the presence of a `Dockerfile` in the sample's root directory. (Make sure you manually build it first)
+    - In the `<sample-name>.yaml` file, ensure the `imageVersion` is provided.
+    - Ensure the following additional files are included, depending on the component type:
+        - For **services**, include an `endpoints.yaml` file in the `/.choreo` directory of the component. ([Template](https://wso2.com/choreo/docs/develop-components/manage-component-source-configurations/#overview-of-the-endpointsyaml-file))
```

Comment:
It's ok to have `endpoints.yaml` for now. This should be replaced with `component.yaml` when it's available. I'll keep track on this.